### ISSUE TITLE
Cannot load `geeup`

### ISSUE THREAD

**Issue by rasenior** [samapriya/geeup#32]
I have followed the instructions [here](https://pypi.org/project/geeup/) exactly, and in various ways. The installation works fine, but when I run `geeup -h` I get a number of error messages that I don't understand. For context, I am using a conda environment ('ee') set up & authenticated for Google Earth Engine. My GDAL installation is from OSGeo4W, so it is installed to 'C:\OSGeo4W64\bin', and is in the environment path. `gdalinfo` works as expected.

The main error messages seem to be:
`command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2`
And:
`A GDAL API version must be specified. Provide a path to gdal-config using a GDAL_CONFIG environment variable or use a GDAL_VERSION environment variable.`

Though there's so much red text in the output that it's difficult for me to see where the problems are really coming from. I've copied the full command prompt input & output [here](https://github.com/samapriya/geeup/files/5886002/cmd_log.txt) because I'm not sure which bits are relevant. Please let me know if there is anything specific that you need to help me identify the problem. 

I'm sure that I had `geeup` set up and working in the past, but I have since removed and re-created this conda environment. By the way, I also use `geeadd` and that has worked great. Thanks!

**Comment by samapriya** [comment]
Hi @rasenior 
I have been able to replicate this.  Seems the issue is happening owing to the changes in compatibility for GDAL and Fiona. If you do have an alternative OS would you mind trying it there and reporting back? I will focus on this but it might take me a week or two.

**Comment by rasenior** [comment]
Thanks, @samapriya. I tried it today on the only other machine I have. Also Windows 10, but it worked! The version of the one that works is 10.0.19041.746, and the one that doesn't is 10.0.18363.1316, if that's relevant at all. The other thing to note is that this machine did not have GDAL pre-installed, but it seems to have been able to do that when running `geeup -h`. That was not the case on the other machine, even when I uninstalled the existing GDAL installation.
